lib/links.ts

```diff
@@ -116,12 +114,15 @@ const openLink = (accountId, shortcut) => {
     return;
   }
 
-  open(match.url, { url: true });
-  logger.success(`We opened ${match.url} in your browser`);
-};
+  const baseUrl = getHubSpotWebsiteOrigin(
+    getEnv() === 'qa' ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD
+  );
+
+  open(`${baseUrl}/${match.getUrl(accountId)}`, { url: true });
+  logger.success(`We opened ${match.getUrl(accountId)} in your browser`);
+}
 
 module.exports = {
-  getSiteLinks,
```

Comment:
Do we need to keep this around in case something was importing it with `require`?